webapp/src/components/expanded_view/component.tsx

```diff
@@ -1349,7 +1349,12 @@ const ExpandedViewGlobalsStyle = createGlobalStyle<{ callThreadSelected: boolean
 
         display: flex;
 
+        > .announcement-bar {
+          display: none;
```

Comment:
This is just to make extra sure. Making the wrapper absolutely positioned is enough to have the view cover everything.